### PR TITLE
Add txt2tags and change manpages of some programs

### PR DESCRIPTION
**Pull request by meator**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

#### Does it build and run successfully? 
(Please choose at least one native build and, if supported, at least one cross build. More are better.)
- [x] I built this PR locally for my native architecture, (x86_64-glibc)
- [x] I built this PR locally for these architectures (if supported. mark crossbuilds):
  - [ ] aarch64-musl
  - [x] armv7l
  - [ ] armv6l-musl
---
This pull request adds the `txt2tags` package, adds a manpage and a missing file to `grml-zsh-config` and removes the pre-generated manpage in `xdgmenumaker`.

The `txt2tag` package was supposed to be added, but [was rejected](https://github.com/void-linux/void-packages/pull/11960#discussion_r287600511) because it only had a python2 version at the time. There is a python3 version now, so I want to add it here. The `xdgmenumaker` package has to include a precompiled version of its manpage in `files/`, because the `txt2tags` package isn't in the repo (more info [here](https://github.com/void-linux/void-packages/pull/11960#issuecomment-496066069)). This pull request removes the premade manpage. And it adds the manpage of `grml-zsh-config`, which depends on `txt2tags`, into the package. It also adds the `etc/zsh/keephack` file to `grml-zsh-config`.

### CODE REVIEW

**Comment by meator** [comment]
I have no idea why the x86_64-musl check fails. The package builds successfully, the `do_check` is the problem. I can not do the check locally in crossbuild. I will look into it later.

**Comment by meator** [comment]
I fixed it. Thanks @paper42!

**Review by Chocimier — COMMENTED** [review]
Thanks for following up, removing prebuild files from repo is a gain.

File: srcpkgs/txt2tags/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+short_desc="Tool for converting plain text into many markup languages"
+maintainer="meator <coder64@protonmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://txt2tags.org/"
+distfiles="https://github.com/txt2tags/txt2tags/archive/refs/tags/${version}.tar.gz"
+checksum=ba745fc1aa98996578afadeb325173f7032a40f161dc4f6f7a7aa08848d11d03
+
+do_check() {
+	tox
```
Comment (Chocimier) [inline srcpkgs/txt2tags/template:22]: Looks line useful tests are skipped that way, and only useless are run. Please rather do following, without any checkkdepends:

    do_check() {
    	vsed -i test/lib.py -e 's:TXT2TAGS = ".*:TXT2TAGS = "build/lib/txt2tags.py":'
    	chmod u+x build/lib/txt2tags.py
    	python3 test/run.py
    }

Comment (meator) [reply srcpkgs/txt2tags/template:22]: Ok, thanks!

**Comment by meator** [comment]
Should I leave the revision of `xdgmenumaker` as is?

**Comment by Chocimier** [comment]
Manpage changed minimally, it's okay to revbump.

**Pull request merged**